server/build/webpack.js

```diff
@@ -248,15 +247,11 @@ export default async function getBaseWebpackConfig (dir, {dev = false, isServer
       new webpack.DefinePlugin({
         'process.env.NODE_ENV': JSON.stringify(dev ? 'development' : 'production')
       }),
-      !isServer && new CombineAssetsPlugin({
-        input: ['manifest.js', 'react.js', 'commons.js', 'main.js'],
-        output: 'app.js'
-      }),
       !dev && new webpack.optimize.ModuleConcatenationPlugin(),
       !isServer && new PagesPlugin(),
       !isServer && new DynamicChunksPlugin(),
       isServer && new NextJsSsrImportPlugin({ dir, dist: config.distDir }),
-      !isServer && new webpack.optimize.CommonsChunkPlugin({
+      dev && !isServer && new webpack.optimize.CommonsChunkPlugin({
```

Comment:
Remove this completely. We are removing the react bundle too.
So, there's no need this in `dev`.

Reply:
Done